pyproject.toml

```diff
@@ -0,0 +1,168 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "newa"
+dynamic = ["version"]
+description = "newa"
+readme = "README.md"
+# TODO: provide correct values
+license = "Apache-2.0"
+license-files = { paths = ["LICENSE.txt"] }
+requires-python = ">=3.9"
+authors = [
+    { name = "Karel Srot", email = "ksrot@redhat.com" },
+    { name = "Patrik Kis", email = "pkis@redhat.com" },
+    { name = "Karel Srot", email = "ksrot@redhat.com" },
```

Comment:
Looks like a duplicate of the first line? Similar below.

Reply:
Fixed in a6ec587